addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -439,6 +439,7 @@ def test_mailing_cron_trigger(self):
             for (test, truth) in [(False, now), (then, then)]:
                 with self.subTest(schedule_date=test):
                     with self.capture_triggers(cron_id) as capt:
+                        common_mailing_values['name'] = f'Knock knock {test}'
```

Comment:
Well then it's not really a common value is it? :D

I'd just put this in the create after `common values`, and remove "Knock knock" from `common values`